src/audiomixerboard.cpp

```diff
@@ -807,13 +816,32 @@ void CAudioMixerBoard::ChangeFaderOrder ( const bool        bDoSort,
     }
 }
 
+QString CAudioMixerBoard::GetTitle()
+{
+    QString myTitle = "";
+    if ( eRecorderState == RS_RECORDING )
+    {
+        myTitle = "[" + tr ( "RECORDING ACTIVE" )  + "] ";
+    }
+    return myTitle + tr ( "Personal Mix at: " ) + strServerName;
+}
+
+void CAudioMixerBoard::SetRecorderState ( const ERecorderState newRecorderState )
+{
+    eRecorderState = newRecorderState;
+    if ( !strServerName.isEmpty() && !bNoFaderVisible )
```

Comment:
I have changed this in my "merge fix". Reason: The recorder state message cannot be received before the new connected client list so this check is not necessary. And if the server name is empty, we still should show the recorder state. Do you agree?

Reply:
I just saw in the code that indeed, the recorder state can be received before the connected client list. Anyway, it does not hurt to show the recorder state even if we do not yet have received the connected client list. What happens is that for a very short time you first see only the recorder state and then you see the recorder state plus the server title. So everything should be fine.

Reply:
Hm... it should work.  If you connect to a non-existent server and have the "trying to connect" title, it's not going to get a recording status message...

(In the previous version of my code, at one point I was setting "Unknown" as the status and had added it to the "trying to connect", then realised what the bNoFaderVisible check did, so included it.  It shouldn't be necessary in this version.)

Reply:
Ok, thanks for the confirmation. Do you have plans to use that new code for your drealm server (which is recording all the time) so that we can test the implementation?

Reply:
Yes, just about to deploy it.

Reply:
Done.